scripts/test_macro_generator.cpp

```diff
@@ -0,0 +1,31 @@
+struct person {
+  std::string name;
+  int age;
+};
+
+
+int a = 20;
+int b = 30;
+
+struct two
+{
+  std::string name;
+  one_t one;
+  int age;
+};
+
+char *iguana = NULL;
+
+struct composit_t { int a; std::vector<std::string> b; int c; std::map<int, int> d; std::unordered_map<int, int> e; double f;};
+
+char *iguana_test = NULL;
```

Comment:
It's unused.

Reply:
Insert the normal statement for test.I will delete it.

Reply:
ok，good，keep it，but comment why you add this unused code.